yarn-project/yarn.lock

```diff
@@ -1251,6 +1251,7 @@ __metadata:
     "@types/node": "npm:^22.15.17"
     jest: "npm:^30.0.0"
     jest-mock-extended: "npm:^4.0.0"
+    json-stringify-deterministic: "npm:1.0.12"
```

Comment:
Do we need this? It seems to be a bit of an obscure unmaintained package - we should keep PXE's dependencies watertight.